example-config.el

```diff
@@ -0,0 +1,31 @@
+;;; example-config.el -- Example Rational Emacs user customization file -*- lexical-binding: t; -*-
+
+;;; Commentary:
+;;
+;; Rational Emacs supports user customization through a `config.el' file
+;; similar to this one.  You can copy this file as `config.el' to your
+;; Rational Emacs configuration directory as an example.
+;;
+;; In your configuration you can set any Emacs configuration variable, face
+;; attributes, themes, etc as you normally would.
+;;
+;; See the README.org file in this repository for additional information.
+
+;;; Code:
+(require 'rational-defaults)
+(require 'rational-screencast)
+(require 'rational-ui)
+(require 'rational-editing)
+(require 'rational-evil)
+(require 'rational-completion)
+(require 'rational-windows)
+
+;; Set further font and theme customizations
+(set-face-attribute 'default nil
+                  :font "JetBrains Mono"
+                  :weight 'light
+                  :height 185)
+
+(load-theme 'doom-snazzy t)
```

Comment:
This may or may not work as expected since we explicitly set the foreground and background colors on the `default-frame-alist`. See #37 for discussion and resolution.

Reply:
Ouch, this shouldn't be necessary to theme things correctly.  Maybe we should remove the default background color setting?  The thing I'm trying to avoid with that is a flash of white when the user starts up the configuration, but maybe the current fix is more problematic than necessary to accomplish it...

Reply:
If we need `custom-file' loaded early, then we probably need to move that into `early-init.el' from beginning of `init.el'  too.

Or would that be a problem? It shouldn't be, should it?